app/src/ui/rebase/rebase-flow.tsx

```diff
@@ -73,14 +73,6 @@ interface IRebaseFlowProps {
 
 /** A component for initiating and performing a rebase of the current branch. */
 export class RebaseFlow extends React.Component<IRebaseFlowProps> {
-  public constructor(props: IRebaseFlowProps) {
-    super(props)
-
-    this.state = {
-      userHasResolvedConflicts: false,
-    }
```

Comment:
wooottt!

Was wondering why this didn't fail and just stumbled across this TypeScript weirdness around empty objects:

https://www.typescriptlang.org/play/#code/DYUwLgBAZg9jEF4IG8C+BYAUF28kEYAmLHAVwDsBjMASxnOgsoAoAjAQwCcAuFVASj4lMUJs1z8yVZkUkixAcgVyp1Og1FVCbLrwDyrAFYhqgtMM2VtE1dtm3mSlfKq16jKgGYdPCDCMmYGYY2C6U3jZh3vZRjspYQA

Do the new linting rules complain about types being empty objects?

Reply:
I believe I caught this when I turned on `@typescript-eslint/ban-types` but I had to turn it off because it was begin triggered a lot. It's probably the rule I'd most want to turn on in the future.